Formula/n/node.rb

```diff
@@ -33,14 +33,24 @@ class Node < Formula
   uses_from_macos "zlib"
 
   on_macos do
-    depends_on "llvm" => [:build, :test] if DevelopmentTools.clang_build_version <= 1100
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1500
   end
 
+  on_linux do
+    depends_on "gcc" if DevelopmentTools.gcc_version("/usr/bin/gcc") < 12
```

Comment:
Will remove conditional as possible that bottle won't work with GCC 12-13.
```suggestion
    depends_on "gcc"
```

More accurate (but not expressible) would be to have separate bottle pour dependency
```suggestion
    depends_on "gcc" => :build if DevelopmentTools.gcc_version("/usr/bin/gcc") < 12
    depends_on "gcc" => <pour-if-insufficient-system-gcc-abi>
```